File: PHP/Iniciante/1006.php
<?php

$a = floatval(readline());
$b = floatval(readline());
$c = floatval(readline());
$peso1 = 2;
$peso2 = 3;
$peso3 = 5;

$media = number_format((($a * $peso1) + ($b * $peso2) + ($c * $peso3)) / 10, 1, '.', '');

echo "MEDIA = {$media}\n";

?>

File: C#/Iniciante/1006.cs
using System; 

class URI {

    static void Main(string[] args) { 

        double a = Convert.ToDouble(Console.ReadLine());
        double b = Convert.ToDouble(Console.ReadLine());
        double c = Convert.ToDouble(Console.ReadLine());    
        
        int peso1 = 2;
        int peso2 = 3;
        int peso3 = 5;
        int soma_pesos = peso1 + peso2 + peso3;
        
        double media = ((a * peso1) + (b * peso2) + (c * peso3)) / soma_pesos;
        
        Console.WriteLine($"MEDIA = {media:F1}");

    }

}
